Makefile

```diff
@@ -52,37 +51,75 @@ test-unit-debug:	build lair
 	LOG_LEVEL=silly NODE_ENV=test npx mocha $(MOCHA_OPTS) ./tests/unit/
 	make stop-lair
 lair:
-	mkdir -p ./tests/tmp
-	rm -rf ./tests/tmp/*
-	mkdir -p ./tests/tmp/shim
-	RUST_LOG=trace lair-keystore --lair-dir tests/tmp/keystore &> hc-lair.log &
+	# Not clearning because on setup we generate keys for the host agents, deleting this file deletes all the keys
+	#rm -rf ./script/install-bundles/keystore
+	mkdir -p ./script/install-bundles/shim
+	rm -rf ./script/install-bundles/shim/*
+	RUST_LOG=trace lair-keystore --lair-dir ./script/install-bundles/keystore &> hc-lair.log &
 stop-lair:
 	killall lair-keystore &
 
+shim:
+	node script/test-shim-init.js &
+
+setup-conductor:
+	make lair
+	sleep 5
+	mkdir -p ./script/install-bundles/shim
+	rm -rf ./script/install-bundles/shim/*
+	make shim
+	sleep 1
+	rm -rf ./script/install-bundles/.sandbox
+	cd script/install-bundles && cargo run
+	make stop-lair
 conductor:
-	RUST_LOG=debug npx holochain-run-dna -c ./tests/app-config.yml -a 4444 -r ./tests/tmp -k shim &> hc-conductor.log &
+	cd script/install-bundles && hc sandbox -f=4444 run -l -p=42233 > ../../hc-conductor.log 2>&1 &
 stop-conductor:
 	yarn run stop-conductor
+	yarn run stop-hc
 
 test-integration:	build DNAs
+	make stop-conductor
 	make stop-lair
-	make lair
-	yarn run stop-conductor
+	make setup-conductor
 	NODE_ENV=test npx mocha $(MOCHA_OPTS) ./tests/integration/
 test-integration-debug:	build DNAs stop-lair lair
-	yarn run stop-conductor
+	make stop-conductor
+	make setup-conductor
 	LOG_LEVEL=silly CONDUCTOR_LOGS=error,warn NODE_ENV=test npx mocha $(MOCHA_OPTS) ./tests/integration/
 
 test-e2e:		build DNAs dist/holo_hosting_chaperone.js
-	yarn run stop-conductor
+	make stop-conductor
+	make setup-conductor
 	NODE_ENV=test npx mocha $(MOCHA_OPTS) ./tests/e2e
 test-e2e-debug:		build DNAs dist/holo_hosting_chaperone.js
-	yarn run stop-conductor
+	make stop-conductor
+	make setup-conductor
 	LOG_LEVEL=silly NODE_ENV=test npx mocha $(MOCHA_OPTS) ./tests/e2e/
 test-e2e-debug2:	build DNAs dist/holo_hosting_chaperone.js
-	yarn run stop-conductor
+	make stop-conductor
+	make setup-conductor
 	LOG_LEVEL=silly CONDUCTOR_LOGS=error,warn NODE_ENV=test npx mocha $(MOCHA_OPTS) ./tests/e2e/
 
+# test-e2e-ci:		build DNAs dist/holo_hosting_chaperone.js
+# 	make e2e-1
+# 	make e2e-2
+# e2e-1:
+# 	make clean-tests
+# 	make stop-conductor
+# 	make setup-conductor
+# 	NODE_ENV=test npx mocha $(MOCHA_OPTS) ./tests/e2e/test_chaperone.js
+# e2e-2:
+# 	make clean-tests
+# 	make stop-conductor
+# 	make setup-conductor
+# 	NODE_ENV=test npx mocha $(MOCHA_OPTS) ./tests/e2e/test_signing.js
+
```

Comment:
Delete this commented code?
```suggestion
```

Reply:
Yes, This was just to test some things out on the CI